README.md

```diff
@@ -101,6 +101,21 @@ Xlsxtream::Workbook.new(io, columns: [
 ])
 # The :columns option can also be given to write_worksheet, so it's
 # possible to have multiple worksheets with different column widths.
+
+# Create the .xlsx file in tmp_file
+# Next change the font name and size
+# Write new datas in the file
+# Allow the file to be downloaded
+tmp_file = Tempfile.new(["FileNameHere", ".xlsx"])
+Xlsxtream::Workbook.open(tmp_file.path, font: { name: 'Arial', size: 11 }) do |xlsx|
+  xlsx.write_worksheet 'Sheet1' do |sheet|
+    sheet << ['Row A1', 'Row B1']
+    sheet << ['Row A2', 'Row B2'] 
+  end 
+end
+  
+send_file tmp_file,
+  type: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
```

Comment:
Is the purpose to show how Xlsxtream is to be used with rails, or something else? (fonts, etc)

If the purpose is to show how the lib is used with Rails I'd suggest an expanded example (mention that it goes into a controller, give an example with/without streaming, etc)

Reply:
This in an example how Xlsxtream can be used with rails Ruby.

Reply:
Alright, if so I'd label that more clearly + drop the stuff that isn't specific to rails (fonts, etc) to make the example cleaner.

Let's see what @felixbuenemann says too, he may have other wishes!

Reply:
Alright.

Reply:
I agree with @sandstrom.

I think the best idea is to add an example rails controller, eg. `examples/rails_controller_tempfile.rb` which shows the usage with a tempfile and we could then add another example like `examples/rails_controller_streaming.rb` later, which shows how to use Xlsxtream with the rails streaming api.

This has already been discussed in #23.